README.md

```diff
@@ -85,7 +85,7 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 20. The admin will review the PR and say **Congrats** if everything looks good.
 
 ## Contributors
-
```

Comment:
why removed this line?

Reply:
> I made a mistake. I fixed this.

Reply:
generally, conversations aren't resolved automatically but manually. but you can see `Outdated` badge if the mentioned code part had been changed